kubecost-exporter.md

```diff
@@ -0,0 +1,117 @@
+# Running Kubecost as a Prometheus metric exporter
+
+Running Kubecost as a Prometheus metric exporter allows you to export various cost metrics to Prometheus without setting up any other Kubecost dependencies. Doing so lets you write PromQL queries to calculate the cost and efficiency of any Kubernetes concept, e.g. namespace, service, label, deployment, etc. You can also calculate the cost of different Kubernetes resources, e.g. nodes, PVs, LoadBalancers, and more. Finally, you can do other interesting things like create custom alerts via AlertManager and custom dashboards via Grafana. 
+
+## Installing
+
+> Note: all deployments of Kubecost function as a Prometheus metric exporter. We strongly recommend helm as an install path to take advantage of Kubecost’s full potential. [View recommended install](http://docs.kubecost.com/install).
+
+If you would prefer to not use the recommended install option and just deploy the Kubecost open source cost model as a metric exporter, you can follow these steps:
+
+
+1. Apply the combined YAML:
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/kubecost/cost-model/develop/kubernetes/exporter/exporter.yaml --namespace cost-model
+    ```
+
+    > If you want to use a namespace other than `cost-model`, you will have to edit the `ClusterRoleBinding` after applying the YAML to change `subjects[0].namespace`. You can do this with `kubectl edit clusterrolebinding cost-model`.
```

Comment:
I think most users would just modify the YAML before applying. I personally think it's ok to drop this in open source docs.

Reply:
My first instinct when following one of our other install instructions was to just modify `kubectl apply` with a different value for the `--namespace` flag.  I ran into this issue and it took me a while to fix it because everything was running but the `cost-model` pod was constantly logging failures (while staying up) because it wasn't getting the right permission scopes and I had to track down why.

Basically, I included this instruction because I would have benefited from it. Definitely interested in a better place for it, because right now I think it visually clutters up the "easy 2 step process" idea.